### ISSUE TITLE
Sync fetch errors

### ISSUE THREAD

**Issue by Jsewill** [ubuntu-touch-oneplus-one/ubuntu-touch-for-oneplus-one#2]
Running repo sync after adding the roomservice.xml to .repo/local_manifests/ produces fetch errors as follows:

    fatal: Couldn't find remote ref refs/heads/cm-11
    error: Cannot fetch ubuntu-touch-oneplus-one/proprietary_vendor_oneplus
    error: Cannot fetch ubuntu-touch-oneplus-one/android_device_oneplus_bacon
    
    error: Exited sync due to fetch errors

  Even with those fetch errors, a much of the sync actually completes.

**Comment by mariogrip** [comment]
I added OnePlusAOSP instead of cm

**Comment by mariogrip** [comment]
The newest update i did should replace kernel files to work with OPO

**Comment by Jsewill** [comment]
Awesome.  Is there anything else I need to do once sync'd, such as run the device/oneplus/bacon/extract-files.sh?

  I get a make error ("make: *** No rule to make target 'build/core/generate_extra_images.mk'") among some warnings after running the following.

        $ repo sync -j 8
        $ . build/envsetup.sh
        $ make -j 8

  I'm just about to comment out the line in device/oneplus/bacon/AndroidBoard.mk that includes generate_extra_images.mk to see if I can get any further.

**Comment by mariogrip** [comment]
That latest push worked for me! It looks like it works, downloading Android 4.4.3 it is required by ubuntu touch...

**Comment by mariogrip** [comment]
you need extract-files, remove any apk  and uncommet any files it cannot fetch for you device.

**Comment by Jsewill** [comment]
In that case, should my repo init command be something like this?

       repo init -u https://code-review.phablet.ubuntu.com/p/aosp/platform/manifest.git -b phablet-4.4.3

Or should I be pulling from a different repository?  Either way, the readme could use an update.

**Comment by mariogrip** [comment]
This one is the correct one: 

repo init -u https://code-review.phablet.ubuntu.com/p/aosp/platform/manifest.git -b phablet-4.4.2_r1

**Comment by Jsewill** [comment]
Hm...okay, that's what I did.  I'll see if running the extract_files script helps any.

**Comment by mariogrip** [comment]
did you run "repo sync" after editing local_manifests?

**Comment by Jsewill** [comment]
Do I have to have AOSP already on the device for the extract_files.sh to work?  I'm getting:

        remote object '/system/vendor/lib/libsecureui.so' does not exist

  I have stock at the moment and flashing the official AOSP caused a boot loop every time I tried it.

  Yes, I ran repo sync every manifest change, and even a few redundant times to make sure I did it, heh.

**Comment by mariogrip** [comment]
open proprietary-files-qc.txt in device/oppo/msm8* and comment out all that does not exist

**Comment by Jsewill** [comment]
Alright.  I did that the last time I attempted to build android for this device, but I wasn't sure that's what I should do.  Hopefully next time I come to this issue it will be to close it.

**Comment by mariogrip** [comment]
Just ask if you need help, i'm gladly to help :)

**Comment by mariogrip** [comment]
Humm, maybe you have some ideas, i flashed boot and recovery. then i started recovery (ubuntu touch version) and the did this ./rootstock-touch-install path/to/trusty-preinstalled-touch-armhf.tar.gz /path/to/system.img as ubuntu wiki says. but when it reboots after install it just goes back to recovery... do you have any ideas why this is like this...

**Comment by Jsewill** [comment]
Unfortunately I haven't even made it that far.  I'd suggest looking for logs on the filesystem, it might tell you if there was a kernel panic or somesuch and why.  The ubuntu touch documentation seems to be very deprecated.

  Well, I've made it a little further, but it looks like I'm going to have to edit some of the source, at least based on the errors:

        system/core/init/property_service.c: In function 'handle_property_set_fd':
        system/core/init/property_service.c:431:10: error: 'PROP_MSG_GETPROP' undeclared (first use in this function)
        system/core/init/property_service.c:431:10: note: each undeclared identifier is reported only once for each function it appears in
        system/core/init/property_service.c:447:10: error: 'PROP_MSG_LISTPROP' undeclared (first use in this function)

**Comment by Jsewill** [comment]
I believe it would be ideal if you can find and use a vivd-preinstalled-touch-armhf tarball.

**Comment by mariogrip** [comment]
I tried that and saucy.

**Comment by mariogrip** [comment]
Ah, so i shuld try utopic then?

**Comment by Jsewill** [comment]
Yeah.  I'm pretty sure that'll be our best bet.

**Comment by Jsewill** [comment]
It appears that my errors are due to "ubuntu/libhybris/hybris/include/hybris/properties/properties.h" is not being included, but I don't know why/how it wouldn't be.

**Comment by mariogrip** [comment]
I did not get any error using my last manifest. try make bootimage then systemimage

**Comment by mariogrip** [comment]
Still no luck with utopic... here is my images https://dl.dropboxusercontent.com/u/56653875/UbuntuTouch-bacon-test.tar.gz (boot, recovery and system) if you want to try..

https://wiki.ubuntu.com/Touch/Install_UT_on_android4.4.2
https://wiki.ubuntu.com/Touch/Building#Set_up_your_development_environment

**Comment by Jsewill** [comment]
I'm going to keep trying to compile it.  I didn't realize I should be running make bootimage and systemimage.  I ran into some errors attempting it, now I'm attempting it again after using https://github.com/janimo/phablet-porting-scripts/blob/master/kernel/check-config to reconfigure the kernel.

  If all else fails I'll try the images you've compiled.  Were you able to find a utopic preinstalled-armhf image?  The closest I could find to utopic is here: http://cdimage.ubuntu.com/ubuntu-touch/ubuntu-rtm/14.09/daily-preinstalled/pending/

**Comment by mariogrip** [comment]
here http://mirror.tversu.ru/ubuntu-cdimage/ubuntu-touch/daily-preinstalled/current/

**Comment by mariogrip** [comment]
![image](https://cloud.githubusercontent.com/assets/1642635/6192497/fceef4a0-b3b2-11e4-9442-b0d2c41c2cbc.png)
I cannot find where the files is after install, this is probably a issue! maybe everything works, just that the image does not install...

**Comment by Jsewill** [comment]
If I remember correctly, they should be somewhere beneath out/target/product/bacon/ .

**Comment by mariogrip** [comment]
no, i mean on the device, i'm using adb shell from recovery

**Comment by Jsewill** [comment]
Ah, I see.  Which image doesn't install?

**Comment by mariogrip** [comment]
i ran rootstock-touch-install with utopic and sysemimage.img

**Comment by mariogrip** [comment]
I started a irc ##UbuntuTouchForOPO join if you want :)
http://webchat.freenode.net/

**Comment by Jsewill** [comment]
This issue was resolved.  I probably should have started another issue for the other things.  Sorry about that.

**Comment by mariogrip** [comment]
btw, i got ubuntu running :) https://forums.oneplus.net/threads/wip-ubuntu-touch-for-oneplus-one.266170
